eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -189,6 +195,16 @@ stages:
                           node ./eng/tools/versioning/increment.js --artifact-name ${{ artifact.name }} --repo-root .
                         displayName: Increment package version
 
+                      - bash: |
```

Comment:
Lets put this in an independent job and PR from the version increment.

Reply:
@weshaggard interesting, why multiple PRs? I am afraid this might not scale well when we add more stuff to be automated after a release is made.

Reply:
My thought is that they are disconnected enough that we shouldn't tie the version increment and the sample updates together. If you guys feel that they are related and should be together and won't block version incrementing than I leave it up to you guys. If you do end up keeping them as a single PR then I agree we should come up with a different title beyond "increment version".

Reply:
Updated branch and title name to have only one PR as per feedback from JS team.